scripts/tlsGen.sh

```diff
@@ -3,13 +3,14 @@ create_pki() {
         mkdir -p "$BASE_DIR/tls/$1"
 
         echo "Generating private key for $1"
-        docker run -it --rm -v $BASE_DIR/tls:/export frapsoft/openssl ecparam -name prime256v1 -genkey -noout -out "/export/$1/$1.key"
+        docker run -it --rm -v $BASE_DIR/tls:/export nginx openssl ecparam -name prime256v1 -genkey -noout -out "/export/$1/$1.key"
```

Comment:
why are we moving to nginx docker image from https://hub.docker.com/r/frapsoft/openssl?

Reply:
Gennady discovered that `frapsoft/openssl` does not support all the architectures we need. `nginx` does...